rslib/src/backend/sync.rs

```diff
@@ -154,6 +155,15 @@ impl crate::services::BackendSyncService for Backend {
         )?;
         Ok(())
     }
+
+    fn set_custom_certificate(&self, _input: generic::String) -> Result<generic::Bool> {
+        #[cfg(feature = "rustls")]
+        return Ok(generic::Bool::from(
+            self.set_custom_cert(_input.val).is_ok(),
+        ));
+        #[cfg(not(feature = "rustls"))]
```

Comment:
Could there be a cleaner way of doing this? Would be nice if I could exclude the whole function but that won't work since it's declared in sync.proto.

Reply:
The only alternative I can think of is defining a separate version of the function. I don't have a preference either way.